### DISCUSSION TITLE
Pinning the TinyMCE toolbar like in Umbraco 8

### DISCUSSION THREAD

**Discussion by mcl-sz** [umbraco/Umbraco-CMS#14171] [Features and ideas]
In Umbraco 8 (and maybe later), TinyMCE's toolbar was pinned to the top of the screen so that there was no need to scroll up and down when the editor's toolbar was about to disappear. 
Umbraco 11 (and perhaps 9 and 10) this functionality has disappeared.
Is it an idea to bring back this functionality?

**Comment by mistyn8** [comment]
https://www.tiny.cloud/docs/tinymce/latest/toolbar-configuration-options/#toolbar_sticky ??
